schema/jsonschema.go

```diff
@@ -20,8 +24,9 @@ var (
 )
 
 func init() {
-	jsonschema.Loaders["http"] = httploader.Load
-	jsonschema.Loaders["https"] = httploader.Load
+	httploader.Client = &http.Client{
+		Timeout: time.Second * 10,
+	}
```

Comment:
Not sure I follow why this changes? Mind elaborating?

Reply:
the default client does not have a timeout configured https://github.com/santhosh-tekuri/jsonschema/issues/92#issuecomment-1309794888